python/TestHarness/RunParallel.py

```diff
@@ -156,7 +167,20 @@ def returnToTestHarness(self, job_index):
     did_pass = True
 
     output = 'Working Directory: ' + tester.specs['test_dir'] + '\nRunning command: ' + command + '\n'
-    output += self.readOutput(f)
+
+    if tester.specs.isValid('redirect_stdout'):
+      # If the tester enabled redirect_stdout and is using more than one processor
+      if tester.specs['redirect_stdout'] != False \
+         and (self.options.parallel != None and int(self.options.parallel) > 1) \
+         and int(tester.specs['max_parallel']) > 1:
```

Comment:
I was handling the -dumb- case where someone uses the the parallel options with a CPU count of 1 :) I also found a number of tests where max_parallel came back as 1000, even though we were clearly asking for only 1 process.

If there is another variable to use, I didn't see it. I'll go back and try again.